src/styles/utils/_normalized.scss

```diff
@@ -0,0 +1,16 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  text-align: center;
```

Comment:
Nothing will be changed
![image](https://user-images.githubusercontent.com/91411692/223080253-d45d1762-dbfd-4aef-822f-10b4b6b7b704.png)

```suggestion
```